galsim/phase_screens.py

```diff
@@ -485,9 +558,11 @@ def Atmosphere(screen_size, rng=None, **kwargs):
 
     if rng is None:
         rng = galsim.BaseDeviate()
+    rng = rng.duplicate()
```

Comment:
I think I added this because without it, my fft_vs_geom_movie script didn't work right for comparing an atmospheric Fourier and geometric PSF.  The two screens (one with kcrit=inf and one with kcrit=0.2/r0) didn't get synchronized phases.  I guess the proper way to solve this is to call `atm=Atmosphere(..., rng=rng.duplicate())` though, instead of duplicating it inside; so I'm happy to revert this change.